### PR TITLE
feat(autocomplete): optimize some const of autocomplete

### PR DESCRIPTION
**Pull request by Zenquan**
Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/autocomplete/src/index.vue [hunk 1]
```diff
@@ -302,12 +302,13 @@ export default defineComponent({
       const highlightItem = suggestionList[index]
       const scrollTop = suggestion.scrollTop
       const offsetTop = highlightItem.offsetTop
+      const scrollHeight = highlightItem.scrollHeight
```
Comment (JeremyWuuuuu) [inline packages/autocomplete/src/index.vue:305]: Can we change it to
```ts
const { offsetTop, scrollHeight } = highlightItem
```

**Comment by JeremyWuuuuu** [comment]
I left a minor comment you can decide you want to change it or not, I am happy to approve as it is. Please let me know if you have any questions.

**Review by jw-foss — APPROVED** [review]

File: packages/autocomplete/src/index.vue (hunk 1 above)
Comment (Zenquan) [reply packages/autocomplete/src/index.vue:305]: This is great！

Comment (Zenquan) [reply packages/autocomplete/src/index.vue:305]: optimize some const of autocomplete

**Review by Zenquan — COMMENTED** [review]
optimize some const of autocomplete

**Pull request merged**